airbyte-integrations/connectors/source-iterable/unit_tests/test_api.py

```diff
@@ -0,0 +1,172 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import pendulum
+import pytest
+import requests
+import responses
+from airbyte_cdk.sources.streams.http.auth import NoAuth
+from source_iterable.api import Campaigns, CampaignsMetrics, Channels, Events, Lists, ListUsers, MessageTypes, Metadata, Templates, Users
+
+
```

Comment:
Would you mind testing all the streams you added to this PR (assertions on `data_field` value).
If you keep my "dynamic stream class generation" suggestion testing the stream generation function would be enough.

Reply:
all new streams initialization are tested in test_source_streams